testnet/validator upgrade 0.6.0 - 0.7.0.md

```diff
@@ -0,0 +1,168 @@
+# Validator Upgrade Guide for laconic_81337-5 Testnet v0.6.0 -> v0.7.0
+
+This guide assumes you have followed the instructions to crete a systemd service or docker container validator node v0.6.0 and you perform the upgrade on a machine running v0.6.0
+
+It is highly recommended to make the backup of your datadir after you stop v0.6.0 and before you start v0.7.0. Unless you changed your datadir, it should be located in `~/.laconicd`
+
+## Systemd service
+Skip this section if you use docker
+
+This is very similar to building v0.6.0. We keep v 0.6.0 running until v0.7.0 is built and only after the successful build we should replace 0.6.0 binary with v0.7.0. This is to avoid jailing your validator for long downtime.
+The general upgrade plan is the following:
+  1. Install Go v1.19 (we used 1.18 for v0.6.0)
+  2. Remove old copy of the github repository and build directory
+  3. Download the latest laconicd repository and checkout v0.7.0
+  4. Build laconicd binary (but not install in this moment)
+  5. Stop laconicd systemd service
+  6. Install recently built new version of laconicd
+  7. Start laconicd service
+
+You have ~10 minutes after step 5 to complete steps 6 and 7 before your validator is jailed for downtime. Getting jailed for downtime is not a disaster, however would require manual unjailing.
```

Comment:
Would mark it as warning. People might oversee this part

Reply:
agree